src/cpp/desktop/DesktopWebPage.cpp

```diff
@@ -46,13 +46,6 @@ namespace {
 WindowTracker s_windowTracker;
 std::mutex s_mutex;
 
-// NOTE: This variable is static and hence shared by all of the web pages
-// created by RStudio. This is intentional as it's possible that the
-// request interceptor from one page will handle a request, and a _different_
-// page will handle the new navigation attempt -- so both pages will need
-// access to the same data.
-std::map<QUrl, int> s_subframes;
```

Comment:
Given where we're at in 1.4, I wouldn't make such a big change now; use of this map predates the workarounds made for the specific Qt issue, so worry there are other side effects/reasons we might be missing here.

Plus we've never tested reliability of NavigationType in the context of Qt 5.12.8, which this code will still be using in everything but Mac desktop, so hate to open up that can of worms.

Maybe open a separate tracking issue to revisit cleaning this up in juliet-rose when we have a bit more time to validate?